core/server/services/settings/index.js

```diff
@@ -3,12 +3,12 @@
  * A collection of utilities for handling settings including a cache
  */
 const _ = require('lodash'),
-    SettingsModel = require('../../models/settings').Settings,
+    debug = require('ghost-ignition').debug('services:settings:index'),
+    common = require('../../lib/common'),
+    models = require('../../models'),
     SettingsCache = require('./cache'),
     SettingsLoader = require('./loader'),
-    ensureSettingsFiles = require('./ensure-settings'),
-    common = require('../../lib/common'),
-    debug = require('ghost-ignition').debug('services:settings:index');
+    ensureSettingsFiles = require('./ensure-settings');
```

Comment:
Just reordered the requires, it's fine to do that in this PR.

- third party first
- rest